<ui version="4.0" >
 <class>OmgTextFileSplitterBase</class>
 <widget class="QDialog" name="OmgTextFileSplitterBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>File Splitter</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/om_icon_small.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTextBrowser" name="textBrowser" >
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">&lt;span style=" font-size:x-large;">Text File Splitter&lt;/span>&lt;/p>
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This program will split a text file looking like this (Genus species header format):&lt;/p>
&lt;p style=" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">Abisara echerius        Latitude        Longitude&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.875  73.875&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.625  73.875&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.625  74.125&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.375  73.125&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">Acraea  violae  Latitude        Longitude&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.875  72.875&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.875  73.125&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.875  73.375&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.875  73.625&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">                20.875  73.875&lt;/p>
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Note that the lattitude and longitude part is optional. in the Genus species header format.&lt;/p>
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Or this (Comma delimited format):&lt;/p>
&lt;p style=" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">Achalarus,lyciades,-83,42.3&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">Achalarus,lyciades,-83,42.3&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">Aglais,milberti,-120.68,50.8&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">Aglais,milberti,-121.63,50.75&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">Aglais,milberti,-121.63,50.75&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier';">Aglais,milberti,-119.61,50.1&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Input and output paths</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>11</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>Input file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="leFileName" />
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="pbnFileSelector" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Output directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leDirName" />
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="pbnDirectorySelector" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Column order</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>11</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="radLongitudeFirst" >
        <property name="text" >
         <string>Longitude first</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="radLatitudeFirst" >
        <property name="text" >
         <string>Latitude first</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>File Type</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>11</number>
      </property>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="radOpenModeller" >
        <property name="text" >
         <string>openModeller localities</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="radCommaDelimited" >
        <property name="text" >
         <string>Comma delimited</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="radGenusSpeciesHeader" >
        <property name="text" >
         <string>Genus species headers</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>leFileName</tabstop>
  <tabstop>pbnFileSelector</tabstop>
  <tabstop>leDirName</tabstop>
  <tabstop>pbnDirectorySelector</tabstop>
  <tabstop>radLongitudeFirst</tabstop>
  <tabstop>radLatitudeFirst</tabstop>
  <tabstop>radGenusSpeciesHeader</tabstop>
  <tabstop>radCommaDelimited</tabstop>
  <tabstop>radOpenModeller</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OmgTextFileSplitterBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>438</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel" >
     <x>321</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OmgTextFileSplitterBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>514</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel" >
     <x>443</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
